net-im/signal-cli-bin/files/signal-cli-sh-0.7.4.patch

```diff
@@ -0,0 +1,31 @@
+diff -Naur a/bin/signal-cli b/bin/signal-cli
+--- a/bin/signal-cli	2020-10-14 18:39:40.000000000 +0200
++++ b/bin/signal-cli	2020-11-04 09:20:25.397469784 +0100
+@@ -84,13 +84,20 @@
+ 
+ 
+ # Determine the Java command to use to start the JVM.
+-if [ -n "$JAVA_HOME" ] ; then
+-    if [ -x "$JAVA_HOME/jre/sh/java" ] ; then
+-        # IBM's JDK on AIX uses strange locations for the executables
+-        JAVACMD="$JAVA_HOME/jre/sh/java"
+-    else
+-        JAVACMD="$JAVA_HOME/bin/java"
+-    fi
++# This package needs Java 11, but Java 11 is currently not part of the gentoo eselect config system
++# so we must search for matching java binaries
```

Comment:
@gyakovlev could you check that this patch is fine? Other than needing a more precise filename what it does. Could this be handled via eselect?

Reply:
@juippis because java 11 can currently be **_not_** selected with `eselect`, and virtual/jre:11 is masked, this workaround is necessary. eselect offers no java 11.

What filename would you prefer? signal-cli-sh-search-java-bin-0.7.4.patch ?

Reply:
Ah, it's masked, okay. Yeah name it something that describes the path in its filename, `${P}-use-working-java-version.patch` or something.

Reply:
@juippis patch is renamed